typing/ctype.mli

```diff
@@ -29,7 +29,7 @@ module Unification_trace: sig
    (** Scope escape related errors *)
     type 'a escape =
     | Constructor of Path.t
-    | Univ of string option
+    | Univ of type_expr
```

Comment:
Maybe you could add there a comment indicating that this is necessarily a `Tunivar _`.

Reply:
Good point.